src/app/components/wallet-list/wallet-list.component.ts

```diff
@@ -40,16 +40,21 @@ export class WalletListComponent implements OnInit {
   }
 
   openModal(template: TemplateRef<any>, walletId?: string) {
+    this.walletService.currentWallet = walletId;
```

Comment:
I think you should not change the `currentWallet` directly. Use `https://github.com/crypto-com/sample-chain-wallet/blob/master/src/app/services/wallet.service.ts#L198` so that it will trigger the event.

Reply:
ok, i'll change it